src/OrchardCore/OrchardCore.Logging.NLog/WebHostBuilderExtensions.cs

```diff
@@ -13,7 +12,9 @@ public static class WebHostBuilderExtensions
     {
         public static IWebHostBuilder UseNLogWeb(this IWebHostBuilder builder)
         {
-            LayoutRenderer.Register<TenantLayoutRenderer>(TenantLayoutRenderer.LayoutRendererName);
+            LogManager.Setup().SetupExtensions(builder =>
+                builder.RegisterLayoutRenderer<TenantLayoutRenderer>(TenantLayoutRenderer.LayoutRendererName));
+
             builder.UseNLog();
             builder.ConfigureAppConfiguration((context, _) =>
```

Comment:
Notice that `builder.UseNLog()` with NLog.Web.AspNetCore v5, will automatically look for `nlog.config` in the `env.ContentRootPath`. 

So `ConfigureAppConfiguration` could be changed to just:
```
builder.ConfigureAppConfiguration((context, _) =>
{
   var environment = context.HostingEnvironment;
   LogManager.Configuration.Variables["configDir"] = environment.ContentRootPath;
}
```